soccer/Processor.hpp

```diff
@@ -163,7 +165,7 @@ class Processor : public QThread {
 
     QMutex& loopMutex() { return _loopMutex; }
 
-    Radio* radio() { return _radio; }
+    Radio* radio() { return _radio->getRadio(); }
```

Comment:
Is this function used anywhere?

Reply:
In MainWindow.cpp:
https://github.com/RoboJackets/robocup-software/blob/cb819ca747de7c868b50851b678fee1bdc9ce5d8/soccer/ui/MainWindow.cpp#L1034-L1039